### PR TITLE
Add clinvar assertion criteria selection

### PR DESCRIPTION
**Pull request by TereseBo**
This PR is a fix for #3762 replaces the text input for assertion metod citation in the ClinVar form with two new inputs, one select for the assertion method citation database (where PubMed, DOI or pmc are the available options) and one text input for the assertion metod citation id. 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data
2. PreClinVar should be supplied with the correct values in the url of the POST request.
3. The appearance of the downloaded files for variants and cases in a ClinVar submission should not have been altered

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by TB, CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
I've started reviewing this PR but looks like the format in your example variant is changing between main branch (PMID:xxx) and this branch (Pubmed:xxx). 

The strings saved in the field should be exactly the same as in main branch, which is also the format expected by preClinvar

**Review by northwestwitch — CHANGES_REQUESTED** [review]
I've started reviewing this PR but looks like the format in your example variant is changing between main branch (PMID:xxx) and this branch (Pubmed:xxx). 

The strings saved in the field should be exactly the same as in main branch, which is also the format expected by preClinvar

**Comment by TereseBo** [comment]
New image after save on this branch:

image

File: scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html [hunk 1]
```diff
@@ -493,10 +536,14 @@ <h3 class="fs-subtitle">Information provided by filling in these fields will be
...
-    var assertCit = document.getElementById('assertion_method_cit').value;
+  $("#assertion_method_cit_db, #assertion_method_cit_id").change(function(){
+    let synonymsACMG=["pmc:4544753", "PMID:25741868", "DOI:10.1038/gim.2015.30"]
+
+    let assertCitDB = document.getElementById('assertion_method_cit_db').value;
+    let assertCitID = document.getElementById('assertion_method_cit_id').value;
     var choices = {{germline_classif_terms|safe}}; // all germline classification terms
-    if (assertCit.includes("PMID:25741868")){ // populate select with ACMG terms only
+
+    if (synonymsACMG.includes(`${assertCitDB}:${assertCitID}`)){ // populate select with ACMG terms only
```
Comment (northwestwitch) [inline scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html:546]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Looks great! 👍🏻

**Pull request merged**